src/docs/design/design-audit.mdx

```diff
@@ -0,0 +1,210 @@
+import { ImageGrid, RuleTable, TOC } from '../../app/components';
+import Banner from '../../app/assets/design/design-audit/banner.svg';
+import TopSeverity from '../../app/assets/design/design-audit/top-severity.png';
+import HighSeverity from '../../app/assets/design/design-audit/high-severity.png';
+import MediumSeverity from '../../app/assets/design/design-audit/medium-severity.png';
+import LowSeverity from '../../app/assets/design/design-audit/low-severity.png';
+import Feature from '../../app/assets/design/design-audit/feature.png';
+import Slides from '../../app/assets/design/design-audit/ppt.svg';
+import DoDont from '../../app/assets/design/design-audit/do-dont.svg';
+import Educational1 from '../../app/assets/design/design-audit/educational-1.png';
+import Educational2 from '../../app/assets/design/design-audit/educational-2.png';
+
+<!-- audit critique review -->
+
+# UI Design Audits
+
+A design audit is a great way to ensure that your user interface design meets Brightlayer UI’s design standard and lines up with other Brightlayer UI-based applications. This process examines not only branding elements, but also the usability of products.
+
+<ImageGrid
+    images={[Banner]}
+    regularWidth
+    caption={'A typical UI audit will highlight problems and provide design recommendations.'}
+/>
+
+Do not wait until product releases look for design audits. Design audits are not just about the cosmetic alignments, but focused more on the usability side, especially when the product team does not have a professional UI/UX designer. Design audits themselves can take weeks, depending on the scope, and development effort the user interface can take months to address the critical / major issues identified.
```

Comment:
```suggestion
Do not wait until a product is released to look for design audits. Design audits are not just about the cosmetic alignments, but focused more on the usability side, especially when the product team does not have a professional UI/UX designer. Design audits themselves can take weeks, depending on the scope, and the development effort to adjust the user interface afterwards can potentially take months.
```

Reply:
For "Do not wait until a product is released to look for design audits"

- Just throwing ideas here but it could state
"Request design audits in the early stage of product readiness or tangible minimal viable product has been reached."
- or
"Request design audits when the product is a tangible minimal viable product"
- or
"Do not request a design audit prior to a product release"